.travis.yml

```diff
@@ -114,9 +114,11 @@ script:
   - |
     if [[ "${INTEG_TESTS}" == "true" ]]; then
       ./integration/integration_test.sh
-      cd "$HOME/gopath/src/github.com/google/certificate-transparency-go"
+      pushd .
+      cd `go list -f '{{ .Dir }}' github.com/google/certificate-transparency-go`
+      chmod +x ./trillian/integration/integration_test.sh
```

Comment:
Why is this chmod necessary?

Reply:
Oh is this that annoying Go Modules behaviour where they overwrite all permissions on the files with 0400 (https://github.com/golang/go/issues/31481)?

Reply:
yes, indeed it is.  I'm doing my best to see if this is going to cause other issues, but I haven't seen any so far other than this.